clients/unshaded/src/main/java/tachyon/hadoop/AbstractTFS.java

```diff
@@ -382,7 +382,6 @@ public FileStatus getFileStatus(Path path) throws IOException {
    * @see org.apache.hadoop.fs.FileSystem#createFileSystem(java.net.URI,
    *      org.apache.hadoop.conf.Configuration)
    */
-  @Override
```

Comment:
hmm, I found it is declared in `FileSystem` at line 215. What version of hadoop-client are you using?